Extension/src/LanguageServer/extension.ts

```diff
@@ -249,27 +248,42 @@ async function installVsix(vsixLocation: string, updateChannel: string): Promise
                     'Resources', 'app', 'bin', 'code') + '"';
             } else {
                 const vsCodeBinName: string = path.basename(process.execPath);
-                const stdout: Buffer = execSync('which ' + vsCodeBinName);
-                return stdout.toString().trim();
+                try {
+                    const stdout: Buffer = execSync('which ' + vsCodeBinName);
+                    return stdout.toString().trim();
+                } catch (error) {
+                    return undefined;
+                }
             }
         }(platformInfo);
-        if (!vsCodeScriptPath) {
```

Comment:
Why is this removed?  There is a case above that could result in vsCodeScriptPath being undefined.

Reply:
I'll add it back in